### PR TITLE
Adding docs for --aws-instance-type

### PR DESCRIPTION
**Pull request by marians**
Relevant with https://github.com/giantswarm/gsctl/pull/138

### CODE REVIEW

File: content/reference/gsctl/create-cluster.md [hunk 1]
```diff
@@ -83,16 +83,29 @@ In this case it doesn't matter if the definition file has an `owner`, `name`, or
...
 
+Some arguments are specific to the provider used in the installation
+(KVM for bare metal clusters, or AWS as a cloud provider).
+
 - `--file`, `-f`: Definition file path. See [cluster defininition reference](../../cluster-definition/) for details.
 - `--name`: Name of the cluster. Overwrites name given in definition file.
 - `--owner`: Name of the owner organization. Overwrites name given in definition file.
 - `--kubernetes-version`: Kubernetes version to use. Overwrites name given in definition file.
 - `--num-workers`: Number of worker nodes. Cannot be combined with `--file`/`-f`.
+- `--dry-run`: Add this flag (no value expected) to simulate the cluster creation. This is especially useful in combination with the global `--verbose`/`-v` flag, which will display the resulting definition YAML based on any command line argument or definition file input.
```
Comment (puja108) [inline content/reference/gsctl/create-cluster.md]: I didn't really understand the last part `definition YAML based on any command line argument or definition file input.`

Comment (marians) [reply content/reference/gsctl/create-cluster.md]: It means that the user will see a YAML file being printed to the terminal. That file could be used in future calls to `gsctl create cluster -f <file>` to create a cluster. Should that be clarified?

BTW, the `--dry-run` part is not new, it has only been moved.

Comment (puja108) [reply content/reference/gsctl/create-cluster.md]: yes, now that you explained it it's clearer, maybe you can explain it similarly in the docs, maybe other's would also be confused like me

Comment (marians) [reply content/reference/gsctl/create-cluster.md]: I have an idea how to improve that, but it will be a larger change. Different PR.

**Pull request merged**